extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/job/AbstractBaseJob.java

```diff
@@ -8,41 +8,47 @@
  */
 package org.eclipse.smarthome.binding.astro.internal.job;
 
-import org.quartz.Job;
-import org.quartz.JobDataMap;
-import org.quartz.JobExecutionContext;
-import org.quartz.JobExecutionException;
+import java.text.SimpleDateFormat;
+import java.util.HashMap;
+import java.util.Map;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  * Baseclass for all jobs with common methods.
  *
  * @author Gerhard Riegler - Initial contribution
+ * @author Christoph Weitkamp - Removed Quartz dependency
  */
-public abstract class AbstractBaseJob implements Job {
+public abstract class AbstractBaseJob implements Runnable {
     private final Logger logger = LoggerFactory.getLogger(AbstractBaseJob.class);
+    public static final SimpleDateFormat ISO8601_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
```

Comment:
@sjka Does it make sense to tackle the NOT thread-safe SDF problem in a more global manner?

As the ThreadLocal approach seams the best approach performance wise an own eclipse smarthome implementation makes the most sense to me. WDYT ?